backend/src/main/java/com/google/blackswan/mock/SimpleAnomalyGenerator.java

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.blackswan.mock;
+
+import static java.util.stream.Collectors.toMap;
+
+import com.google.models.*;
+import java.util.Scanner;
+import java.util.*;
+import java.text.ParseException;
+import java.util.logging.Logger;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.lang.Math;
+import java.lang.Object;
+
+/** Generate list of anomalies based on data in the csv file using average and threshold. */
+public class SimpleAnomalyGenerator implements AnomalyGenerator {
+  private static final String DATA_FILE_LOCATION = "/sample-ramen-data.csv";
```

Comment:
seems this will be used in `getResourceAsStream(DATA_FILE_LOCATION)`. So I'm assuming this will need to be a relative path, but needs to be in the /resources folder on github?

Reply:
Maven automatically checks for this file in the `src/main/resources` folder, so the current `DATA_FILE_LOCATION` should work even on github. 

Making the `DATA_FILE_LOCATION` a relative path i.e. `./sample-ramen-data.csv` results in maven not being able to find the file in the `resources` folder, since making it a relative path makes maven try to find the file in the directory of the class that is using the csv file. If I change the path to relative, in order to make it work, there would be a lot of `../../../`...

or i can move the file out of the resources folder and into the directory of the java files and make the path relative if you think that is better!

Reply:
resources folder seems fine. thanks for explaniation.